### ISSUE TITLE
1.12.2 - I&F 1.9.1 - Stage 1 Dragon teleports to summon/hatch location when dropped

### ISSUE THREAD

**Issue by SlendyMcTendies** [Alex-the-666/Ice_and_Fire#2609]
Title pretty much says it all. After hatching a fire dragon, every time I drop the dragon from my shoulder it teleports to the location it was summoned from with the dragon horn. It also did this for it's hatching location. It would teleport to the exact square and height position, despite having nothing to actually stand/walk on.

**Comment by Zdiddy0** [comment]
I am having the same problem. This problem seems to only be a concern with stage 1.(?) I made a new world where I hatched a dragon and in stage 1 he was just walking away until I picked him up. After I put him down he wouldn’t move or anything. I used meals to get him to stage 2 and now he is fine. Atleast stage 2+ dragons don’t have this issue.

**Comment by anoukeblack** [comment]
I'm having this same problem. I just hatched a dragon and set its home location a short distance away. Now when I put it down from off my shoulder, it teleports to that spot no matter where I am. It also doesn't wander or move no matter the mode I set it to. Also after releasing it from the horn anywhere but the area that I set as the home location, it disappears and I have to relog and find it back in the horn again.  Good to see at least from the other comments that it stops happening at stage 2, but it's disappointing that I can't have it follow me or set it down anywhere as a stage 1 as I'd love to have it follow me around.
